src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,130 @@
+import classNames from 'classnames';
+import { useConfig } from 'wagmi';
+import { Avatar, AvatarIcon, IconType, NumberFormat, formatterUtils } from '../../../../core';
+import { type ICompositeAddress, type IWeb3ComponentProps } from '../../../types';
+import { AssetTransferAddress } from './assetTransferAddress';
+
+export interface IRequiredCompositeAddress extends ICompositeAddress {
+    /**
+     * Required address the transfer participants.
+     */
+    address: string;
+}
```

Comment:
I would probably update the base `ICompositeAddress` to set the `address` as required, we usually want to have the same handling on all components.
@Fabricevladimir Do you see an use case where we only have the ens name and not the address of the `publisher` of a proposal?

Reply:
Sounds good, thanks @Fabricevladimir 👌🏻 
@thekidnamedkd Could you update the interface and update the proposal-list-item component to reflect such?